crates/ark/src/modules/positron/help.R

```diff
@@ -163,6 +159,31 @@ getHtmlHelpContentsInstalled <- function(helpFiles, package) {
   paste(contents, collapse = "\n")
 }
 
+getPackageNameFromHelpPath <- function(path) {
```

Comment:
This is where having these modules as a little package would be handy, because I wanted to write a few tests for this helper but have no where to put them